test/run_tests

```diff
@@ -111,59 +107,55 @@ argv.exe = os.path.abspath(argv.exe)
+********************************** OpenSn Test system **********************************
 
-Use flag -h to print argument options.
-Test directories identified:""")
+Use -h to print argument options.
 
+Test directories found:""")
 for test_dir in test_hierarchy:
-    print("  " + os.path.relpath(test_dir, argv.directory))
+    print("  " + test_dir)
+
 print()
-print("Description of what you are looking at:")
-print("\033[33m[XX]\033[0m<test directory>/<lua filename>.lua.......", end="")
-print("\033[36m[annotations]\033[0m", end="")
+print("Output description:")
+print("\033[33m[XX]\033[0m <test directory>/<lua filename>.lua (lua args).......", end="")
+print("\033[36m[Annotations]\033[0m", end="")
 print("\033[32mPassed\033[0m/\033[31mFailed\033[0m")
-print("\033[33m[XX]\033[0m = number of mpi processes")
-print("\033[36m[annotations]\033[0m = error messages preventing a test from " +
-      "running")
-print("\033[36m[lua file missing]\033[0m = A test pointing to a .lua file was "
-      "indicated in a .json file but the actual lua file is missing.")
-print("\033[36m[Gold file missing]\033[0m = Test with a GoldFile check has no "
-      ".gold file in the gold/ directory. If the input is Input.lua \n"
-      # [Gold file missing] =
-      "                      then there"
-      " needs to be a Input.lua.gold file in the gold directory. If this was "
-      "the first run \n"
-      "                      then copy Input.lua.out from the out/ directory "
-      "and use that as the gold.")
-print("\033[36m[Python error]\033[0m = A python error occurred. Run with -v 1 "
-      "to see the error.")
+print("\033[33m[XX]\033[0m : Number of MPI processes")
+print("\033[36m[Annotations]\033[0m : Error messages preventing a test from running\n")
+print("\t\033[36m[Skipped]\033[0m")
+print("\tTest was intentionally skipped. A reason for skipping this test should be")
+print("\tprovided by the responsible developer.\n")
+print("\t\033[36m[Lua file missing]\033[0m")
+print("\tA test pointing to a .lua file was found in a .json file but the lua file is")
+print("\tmissing.\n")
+print("\t\033[36m[Gold file missing]\033[0m")
+print("\tTest with a gold file check has no .gold file in the gold/ directory. If the")
+print("\tinput is input.lua then there needs to be an input.lua.gold file in the gold")
+print("\tdirectory. If this was the first run then copy Input.lua.out from the out/")
+print("\tdirectory and use that as the gold file.\n")
+print("\t\033[36m[Python error]\033[0m")
+print("\tA python error occurred. Run with -v 1 to see the error.")
 
 print()
 
-jobmanager.PrintCaughtWarnings(warning_manager1,
-                               "Warnings during test configuration")
+job_manager.PrintCaughtWarnings(warning_manager1, "Warnings during test configuration")
 print()
 
 with warnings.catch_warnings(record=True) as warning_manager2:
-    retval = jobmanager.RunTests(tests, argv)
+    RETVAL = job_manager.RunTests(tests, argv)
```

Comment:
Any specific reason for all-caps? I don't believe there is a coding standard for python, but this seems off.

Reply:
Ugh... that was a pylint thing that I forgot to undo. Fixed.